tests/webTables.spec.ts

```diff
@@ -55,10 +55,10 @@ test.describe('Web Tables', () => {
     test('Test Case 4: Validate phone number and pet name on the Owner Information page ', async ({ page }) => {
         await page.getByRole('button', { name: 'Owners' }).click()
         await page.getByRole('link', { name: 'Search' }).click()
-        const targetRow = page.getByRole('row', { name: '6085552765' })
-        const petName = await targetRow.locator('td').nth(4).textContent()
-        await targetRow.getByRole('link', { name: 'Peter McTavish' }).click()
-        await expect(page.getByRole('cell', { name: '6085552765' })).toContainText('6085552765')
+        const phoneNumber = '6085552765'
+        const petName = await page.getByRole('row', { name: '2387 S. Fair Way' }).locator('td').nth(4).textContent()
```

Comment:
According to test case requirements, you need to locate owner by the phone number: `Locate the owner by the phone number "6085552765". Extract the Pet name displayed in the table for the owner and save it to the variable. Click on this owner.`